docs/howto/determining-packages.md

```diff
@@ -0,0 +1,172 @@
+# Determining Packages in Pangeo Docker Images
+
+For deployments of Pangeo Docker Images using JupyterHub or Binder it can be sometimes unclear to the user exactly which image and tag is being used. Here are the images available as Pangeo Docker Images:
+
+| Image           | Description                                   | 
+|-----------------|-----------------------------------------------|
+| [base-notebook](base-notebook/packages.txt) | minimally functional image for pangeo hubs | 
```

Comment:
Do these links work in the built sphinx documentation?

Reply:
Good point. Updated to point to the corresponding GitHub url instead.